src/airobot/robot/ur5e_pybullet.py

```diff
@@ -0,0 +1,721 @@
+            p.setJointMotorControlArray(self.robot_id,
+                                        self.actuator_ids[:-1],
+                                        p.VELOCITY_CONTROL,
+                                        targetVelocities=tgt_vels,
+                                        forces=forces)
+        else:
+            jnt_id = self.jnt_to_id[joint_name]
+            p.setJointMotorControl2(self.robot_id,
+                                    jnt_id,
+                                    p.VELOCITY_CONTROL,
+                                    targetVelocity=0,
+                                    force=0.0)
+
+    def disable_torque_control(self, joint_name=None):
+        """
+        Disable the torque control mode in Pybullet
+
+        Args:
+            joint_name: If it's none, then all the six joints
+                on the UR robot are disabled with torque control.
+                Otherwise, only the specified joint is disabled with
+                torque control.
+                The joint(s) will enter velocity control mode.
+
+        """
+        if joint_name is None:
+            self.set_jvel([0.0] * 7)
+        else:
+            self.set_jvel(0.0, joint_name)
+
+    def get_jpos(self, joint_name=None):
+        """
+        Return the joint position(s)
+
+        Args:
+            joint_name: If it's None, it will return joint positions
+                of all the actuated joints. Otherwise, it will
+                return the joint position of the specified joint
+
+        Returns:
+            joint position (float) or joint positions (list) depends on
+            the joint_name
+        """
+        if joint_name is None:
+            states = p.getJointStates(self.robot_id, self.actuator_ids)
+            pos = [state[0] for state in states]
+        else:
+            jnt_id = self.jnt_to_id[joint_name]
+            pos = p.getJointState(self.robot_id, jnt_id)[0]
+        return pos
+
+    def get_jvel(self, joint_name=None):
+        """
+        Return the joint velocity(ies)
+
+        Args:
+            joint_name: If it's None, it will return joint velocities
+                of all the actuated joints. Otherwise, it will
+                return the joint velocity of the specified joint
+
+        Returns:
+            joint velocity (float) or joint velocities (list) depends on
+            the joint_name
+        """
+        if joint_name is None:
+            states = p.getJointStates(self.robot_id, self.actuator_ids)
+            vel = [state[1] for state in states]
+        else:
+            jnt_id = self.jnt_to_id[joint_name]
+            vel = p.getJointState(self.robot_id, jnt_id)[1]
+        return vel
+
+    def get_jtorq(self, joint_name=None):
+        """
+        Return the joint torque(s) applied during the last stepSimulation
+        in VELOCITY_CONTROL and POSITION_CONTROL modes.
+        In TORQUE_CONTROL, the applied joint motor torque is
```

Comment:
grammar issues in doc-string. Pass through grammarly.

Reply:
commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50